src/App.tsx

```diff
@@ -30,39 +32,130 @@ function App() {
+    https://cdc.dwd.de/geoserver/CDC/ows?&service=WFS&version=2.0.0&request=GetFeature&typenames=CDC%3AGRD_DEU_API.GET_FEATURE_INFO&outputformat=application/json&viewparams=observed_property_code%3AGRD_DEU_P1D_BF-GRB%3Blocation%3APOINT(8.55+52.06)%3Bsrid%3A4326%3Bbegin_position%3A2023-07-29T00%3A00%3A00Z%3Bend_position%3A2023-08-28T23%3A59%3A59Z%3Bsrid_out%3A4.326%3Bregexp%3Atrue
+    ===============
+    https://cdc.dwd.de/geoserver/CDC/ows?service=WFS&version=2.0.0&request=GetFeature&typeName=CDC:OBSERVATIONS_API.GET_EVENT_RESULT_TIMES&outputFormat=text/javascript&format_options=callback:set_1878732965_Layer_Time&sortBy=TIME_STAMP&viewparams=observed_property_guid:1564838978;begin_position:2022-07-29T00:00:00.000Z;end_position:2023-08-28T00:00:00.000Z&callback=jQuery11020880727675764279_1693298035627&_=1693298035628
+    https://cdc.dwd.de/geoserver/wfs?service=wfs&version=2.0.0&request=GetFeature&typeNames=CDC:OBSERVATIONS_API.GET_EVENT_RESULT_TIMES
+    https://cdc.dwd.de/geoserver/CDC/ows?service=WFS&version=2.0.0&request=GetFeature&typeName=CDC:OBSERVATIONS_API.GET_EVENT_RESULT_TIMES
+
+    BEISPIEL --> SLIDER
+
+
+    https://cdc.dwd.de/geoserver/CDC/wms?service=WMS&version=1.1.0&request=GetMap&layers=CDC%3AGRD_DEU_P1D_BF-GRB
+    &bbox=3280415.0,5237501.0,3934415.0,6103501.0&width=579&height=768&srs=EPSG%3A31467&styles=&format=application%2Fopenlayers3
+    &ELEVATION=10&TIME=2023-07-22T00%3A00%3A00.000Z
 
+    &ELEVATION=10
+    &TIME=2023-08-22T00%3A00%3A00.000Z
+    */
+
+    //RESIZE FEHLT
+
+    const test = () => {
+        console.log(
+            "https://cdc.dwd.de/geoserver/CDC/ows?&service=WFS&version=2.0.0&request=GetFeature&propertyname=OBSERVED_PROPERTY_CODE%2CRESULT_TIME%2CRESULT&typenames=CDC%3AGRD_DEU_API.GET_FEATURE_INFO&outputformat=application/json&viewparams=observed_property_code%3AGRD_DEU_P1D_BF-GRB%3Blocation%3APOINT(8.37+52.07)%3Bsrid%3A4326%3Bbegin_position%3A2023-06-23T00%3A00%3A00Z%3Bend_position%3A2023-07-23T23%3A59%3A59Z%3Bsrid_out%3A4.326%3Bregexp%3Atrue",
+        );
+    };
     return (
-        <div className="App">
+        <div className="App" style={{ zIndex: 1000 }}>
+            {/* <button onClick={test} /> */}
             <SidePanel isOpen={infoOpen} dismissPanel={() => dismissInfo()} children={<Markdown file={"info.md"} />} />
             <SidePanel
                 isOpen={imprintOpen}
                 dismissPanel={() => dismissImprint()}
                 children={<Markdown file={"imprint.md"} />}
             />
+            <SidePanel
+                isOpen={overlayOpen}
+                dismissPanel={() => dismissOverlay()}
+                children={<ControlOverlayUI showLayer={showLayer} setShowLayer={setShowLayer} />}
+            />
             <MapContainer zoomControl={false} center={position} zoom={zoom} style={{ height: height }}>
                 <MapHack setMap={setMap} />
-                <MoistureMarkers />
+                {/*<MoistureMarkers />*/}
```

Comment:
Die hätten wir schon noch gerne. :)
```suggestion
                <MoistureMarkers />
```